src/main/scala/io/iohk/ethereum/keystore/Wallet.scala

```diff
@@ -0,0 +1,9 @@
+package io.iohk.ethereum.keystore
+
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.{Address, SignedTransaction, Transaction}
+
+case class Wallet(address: Address, privKey: ByteString) {
+  def signTx(tx: Transaction): SignedTransaction = ???
```

Comment:
We already have `SignedTransaction.sign` maybe we can overload it with the one that accepts `Wallet`?